setup.cfg

```diff
@@ -46,6 +46,7 @@ install_requires =
     scikit-image
     sidecar>=0.5.1
     ipypopout>=0.0.10
+    astroquery
```

Comment:
is there any minimum version where the SDSS query was introduced or syntax changed to what we need?

Reply:
I have no idea because I wasn't able to find any version. Maybe I wasn't looking in the correct areas?

Reply:
Just went and looked through astroquery, and it looks like SDSS has been there since the first 0.1 release, so it's probably safe to leave without a minimum version for now.